v4/source/introduction.yaml

```diff
@@ -214,23 +214,183 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          event: 'added_to_team',
+          data: {
+            team_id: '<team_id>',
+            user_id: '<user_id>'
+          },
+          broadcast: {
+            omit_users: null,
+            user_id: '<user_id>',
+            channel_id: '<channel_id>',
+            team_id: '<team_id>'
+          },
+          seq: '<seq_id>'
+        }
+        ```
       - authentication_challenge
-      - channel_converted
+        ```
+        {
+          token: <token-id>
+        }
+        ```
       - channel_created
+        ```
+        {
+          'event': 'channel_created',
+          'seq'  : '<seq_id>',
+          'data' : {
+            'channel_id': '<channel_id>',
+            'team_id'   : '<team_id>'
+          },
+          'broadcast': {
+            'omit_users': None,
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          }
+        }
+        ```
       - channel_deleted
-      - channel_member_updated
+        ```
+        {
+          'event': 'channel_deleted',
+          'data': {
+            'channel_id': '<channel_id>'
+           },
+          'broadcast': {
+            'omit_users': 'null',
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'},
+            'seq': '<seq_id>'
+
+          ```
       - channel_updated
+         ```
+         {
+           'event': 'channel_updated',
+           'data': {
+             'channel': {
+               'id': '<channel_id>',
+               'create_at': '<timestamp>',
+               'update_at': '<timestamp>',
+               'delete_at': '<timestamp>',
+               'team_id': '<team_id>',
+               'type': 'O',
+               'display_name':'<channel_id>',
+               'name': '<channel_name>',
+               'header': 'header',
+               'purpose': '',
+               'last_post_at': '<timestamp>',
+               'total_msg_count': '<total_msg_count>',
+               'extra_update_at': '<timestamp>',
+               'creator_id': '<basic_user_id>'
+             },
+           'broadcast': {
+             'omit_users': null,
+             'user_id': '',
+             'channel_id': '<channel_id>',
+             'team_id': '<team_id>'
+           },
+             'seq': '<seq_id>'
+           }
+         ```
       - channel_viewed
+        ```
+        {
+          'desc': '<desc_id>'
```

Comment:
I don't think this event payload is correct. If I open my communtiy.mattermost.com account in Chrome and watch the events on the WebSocket from the network tab in the dev console I see:

```
{"event": "channel_viewed", "data": {"channel_id":"somechannelid"}, "broadcast": {"omit_users":null,"user_id":"myuserid","channel_id":"","team_id":""}, "seq": 3}
```